insonmnia/marketplace/server_test.go

```diff
@@ -258,6 +267,7 @@ func TestInMemOrderStorage_GetOrders_Count(t *testing.T) {
 			Price:     "1",
 			OrderType: pb.OrderType_BID,
 			Slot: &pb.Slot{
+				Duration:  uint64(structs.MinSlotDuration.Seconds()),
```

Comment:
How many such slots do you have?
I wonder if you changed those with some kind of sed or did it by hand :)

Reply:
It made by hand, of course!